layers/meta-balena-allwinner/recipes-kernel/linux/linux_4.19.76.bb

```diff
@@ -0,0 +1,26 @@
+SECTION = "kernel"
+DESCRIPTION = "Mainline Linux kernel with patches from megous"
+
+inherit kernel
+require recipes-kernel/linux/linux.inc
+
+LICENSE = "GPLv2"
+LIC_FILES_CHKSUM = "file://COPYING;md5=bbea815ee2795b2f4230826c0c6b8814"
+COMPATIBLE_MACHINE = "(sun4i|sun5i|sun7i|sun8i)"
+
+# Pull in the devicetree files into the rootfs
+RDEPENDS_kernel-base += "kernel-devicetree"
+
+# Default is to use stable kernel version
+# If you want to use latest git version set to "1"
+DEFAULT_PREFERENCE = "-1" 
+
+KERNEL_EXTRA_ARGS += "LOADADDR=${UBOOT_ENTRYPOINT}"
+
+PV = "4.19.76+git${SRCPV}"
+SRCREV_pn-${PN} = "v4.19.76"
+
+
+SRC_URI = "git://git.kernel.org/pub/scm/linux/kernel/git/stable/linux-stable.git;protocol=git;branch=linux-4.19.y \
+        "
+S = "${WORKDIR}/git"
```

Comment:
Where is this recipe taken from?

Reply:
It's a new recipe based on the linux_4.14%.bb recipe from the same folder.

Reply:
The warrior branch of meta-sunxi has 4.19.63. Why not use that one?

Reply:
We could use the kernel version in the bsp (4.19.63) and just use the kernel patches from armbian on top of it

Reply:
Some patches don't apply:
(227/344) add-BergMicro-SPI-flashes.patch
| [INFO]: check of .kernel-meta//patches///add-BergMicro-SPI-flashes.patch with "git am" did not pass, trying reduced context.
| [INFO]: Context reduced git-am of .kernel-meta//patches///add-BergMicro-SPI-flashes.patch with "git am" did not work, trying "apply".
| (228/344) add-orangepi-zeroplus2.patch
| [INFO]: check of .kernel-meta//patches///add-orangepi-zeroplus2.patch with "git am" did not pass, trying reduced context.
| [INFO]: Context reduced git-am of .kernel-meta//patches///add-orangepi-zeroplus2.patch with "git am" did not work, trying "apply".